integration-tests/script/setup_druid_on_k8s.sh

```diff
@@ -25,13 +26,6 @@ rm -rf docker/client_tls
 cp -r client_tls docker/client_tls
 cd ..
 
-# Build Docker images for pods
-mvn -B -ff -q \
```

Comment:
Lets mention a comment that this command relies on the correct version of the jars be installed in the local m2 repo. For any changes please rebuild it using the command.

Reply:
Have added a comment.